chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1168,7 +1160,13 @@ impl PeerManagerActor {
     ///  - bootstrap outbound connections from known peers,
     ///  - unban peers that have been banned for awhile,
     ///  - remove expired peers,
```

Comment:
add a comment explaining the difference between 'interval', 'default_interval' and 'max_interval

Reply:
Ok, I added an explanation.